metagraph/integration_tests/base.py

```diff
@@ -6,7 +6,12 @@
 
 script_path = os.path.dirname(os.path.realpath(__file__))
 
-METAGRAPH = f'{os.getcwd()}/metagraph'
+USE_GDB = False
```

Comment:
What would be a consistent way to read it from params?

Reply:
You can add a flag here:
https://github.com/ratschlab/metagraph/blob/7535d81511478109a7857c0e93f133948fe3ed00/metagraph/integration_tests/main.py#L33